packages/helldoc-core/src/app/components/NavBar.tsx

```diff
@@ -24,26 +27,28 @@ class NavBar extends React.Component<{
     const navs = resolveNavs();
     return (
       <div className="navbarstyle">
```

Comment:
consider rename `navbarstyle` to `navbar-style` ? Or just rename to 'navbar' enough .